src/error.ts

```diff
@@ -697,6 +697,10 @@ const RETRYABLE_WRITE_ERROR_CODES = new Set<number>([
   MONGODB_ERROR_CODES.ExceededTimeLimit
 ]);
 
+export function isRetryableEndTransactionError(error: MongoError): boolean {
```

Comment:
do we expect to rename this function when we implement the next ticket or add a new function that returns the same thing to address the missing cases? i.e., will this become the new `isRetryableError` check?

Reply:
I believe this will go away as you mention and become `isRetryable` error, barring no other complications.